src/ir_Kelvinator.cpp

```diff
@@ -442,7 +487,7 @@ String IRKelvinatorAC::toString(void) const {
   result += addBoolToString(_.IonFilter, kIonStr);
   result += addBoolToString(_.Light, kLightStr);
   result += addBoolToString(_.SwingH, kSwingHStr);
-  result += addBoolToString(_.SwingV, kSwingVStr);
+  result += addIntToString(_.SwingV, kSwingVStr);
```

Comment:
You should use [`addSwingVToString()`](https://crankyoldgit.github.io/IRremoteESP8266/doxygen/html/namespaceirutils.html#a4921dc8d85aa1cc3e89aaa64c24abd58) instead.
e.g. (Not tested. Estimate only)
```c++
result += addSwingVToString(_.SwingV, kKelvinatorSwingAuto, kKelvinatorSwingUp, kKelvinatorSwingUp, kKelvinatorSwingMiddleUp, kKelvinatorSwingMiddle, kKelvinatorSwingMiddleDown, kKelvinatorSwingMiddleDown, kKelvinatorSwingDown, kKelvinatorSwingLastPos, kKelvinatorSwingAuto, kKelvinatorSwingAuto, kKelvinatorSwingAuto);
```
Also add the following at the top of the file near the others:
```c++
using irutils::addSwingVToString;
```